index.js

```diff
@@ -24,14 +24,23 @@ function forEach(xs, fn) {
 }
 
 // TODO: use object-keys
-var objectKeys = Object.keys || function keys(obj) {
+function objectKeys(obj) {
 	var res = [];
-	for (var key in obj) { res.push(key); } // eslint-disable-line no-restricted-syntax
+
+	for (var key in obj) { // eslint-disable-line no-restricted-syntax
+		res.push(key);
+	}
+
+	if (Object.getOwnPropertySymbols) {
+		for (var symbol in Object.getOwnPropertySymbols(obj)) { // eslint-disable-line no-restricted-syntax
+			res.push(symbol);
+		}
+	}
+
 	return res;
-};
+}
 
-// TODO: use object.hasown
```

Comment:
the todo is for using the package, since relying on the builtin would be a breaking change

Reply:
I can revert this change, since as it currently stands, the fallback function does not work for any use case, since the call arguments are incompatible, so it doesn't interfere with Symbol usage.